python-checks/src/test/resources/checks/django/djangoModelFormFieldsCheck.py

```diff
@@ -0,0 +1,29 @@
+from django import forms
+
+class ExcludeFieldsForm(forms.ModelForm):
+    class Meta:
+        model = MyModel
+        exclude = ['field1', 'field2']  # Noncompliant {{Set the fields of this form explicitly instead of using "exclude".}}
```

Comment:
__Nitpick:__ we could have one assertion on location as well as message, for the sake of completeness.

Reply:
done